src/applets/clock/com.solus-project.clock.gschema.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schemalist gettext-domain="budgie">
+<schema id="com.solus-project.clock">
+    <key type="b" name="clock-show-date">
+      <default>false</default>
+      <summary>Show seconds</summary>
+      <description>Whether to show the date.</description>
+    </key>
+    <key type="b" name="clock-show-seconds">
+      <default>false</default>
+      <summary>Show seconds</summary>
+      <description>Whether to show seconds in the clock.</description>
+    </key>
+    <key type="b" name="clock-use-24-hour-time">
+      <default>false</default>
+      <summary>Use 24 hour time</summary>
+      <description>Whether to use the 24 hour time format.</description>
+    </key>
+    <key type="b" name="clock-use-custom-format">
+      <default>false</default>
+      <summary>Use custom format</summary>
+      <description>Whether to use a custom format for the clock.</description>
+    </key>
+    <key type="s" name="clock-custom-format">
+      <default>"%H:%M:%S %d.%m.%Y"</default>
+      <summary>Format</summary>
+      <description>Format of the clock</description>
+    </key>
+    <key type="b" name="clock-use-custom-timezone">
+      <default>false</default>
+      <summary>Use custom timezone</summary>
+      <description>Whether to use a custom timezone instead of the system one.</description>
+    </key>
+    <key type="s" name="clock-custom-timezone">
```

Comment:
IMO the use of "clock-SETTING" is a bit redundant since it is already within the clock schema. The prefixed `clock-` could be removed.